pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -15,15 +15,17 @@ type flavorRequirements struct {
 }
 
 var (
+	checkStorageFlavor      bool
```

Comment:
Please move the declaration of `checkStorageFlavor` in the context of `ValidateMachinePool` or we'll run into data races if we make those checks concurrent.